src/TodoItem.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import User from './User';
+
+const Todoitem = (props) => {
+  const { id, title, completed, user } = props.todo;
+
+  return (
+    <tr className="App-table-row">
+      <td>{id}</td>
+      <td>{title}</td>
+      <td className={completed ? 'completed' : 'not-completed'}>
+        {completed ? 'completed' : 'not completed'}
+      </td>
+      <User user={user} />
```

Comment:
Три подряд слова user читаются не очень гуд
```suggestion
      <User name={user.name} />
```